bin/renew_tokens

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'bundler/setup'
+require 'fortnox/api'
+require 'dotenv'
+
+DOTENV_FILE_NAME = '.env'
+Dotenv.load(DOTENV_FILE_NAME)
+
+Fortnox::API.configure do |config|
+  config.client_secret = ENV.fetch('FORTNOX_API_CLIENT_SECRET', nil)
```

Comment:
Yeah, this is what I mean ... We should offload this as well to the client application instead of requiring them to configure it via the gem.